src/components/ResultsTable.tsx

```diff
@@ -1,116 +1,205 @@
-import React from 'react'
+import React from "react";
```

Comment:
change your prettier config to use single quotes

Reply:
I've realised we are actually relying on eslint for styling. Can you install eslint in your VS Code and run `npm run eslint -- --fix`? That will do the formatting for you.